### PR TITLE
palatte-namespaces-pods

### PR DESCRIPTION
**Pull request by resmaa**
Added **Palette specific** namespaces and pods to troubleshooting
1. Tenant Cluster NS with Pods
2. PCG NS with Pods
3. Enterprise Cluster NS with Pods

### CODE REVIEW

**Comment by spectro-prow** [comment]
Hi @resmaa. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: packages/docs/content/14-troubleshooting.md
Comment (truegilb) [inline packages/docs/content/14-troubleshooting.md:193]: Let's add some descriptions for the purpose of the tables below.

**Review by truegilb — CHANGES_REQUESTED** [review]

File: packages/docs/content/14-troubleshooting.md
Comment (resmaa) [reply packages/docs/content/14-troubleshooting.md:193]: sure

File: packages/docs/content/14-troubleshooting.md
Comment (truegilb) [inline packages/docs/content/14-troubleshooting.md:213]: duplicate - please check for other places

File: packages/docs/content/14-troubleshooting.md
Comment (truegilb) [inline packages/docs/content/14-troubleshooting.md:228]: Add "Here is a mapping of the pods and namespaces." before presenting the table.

**Review by truegilb — CHANGES_REQUESTED** [review]
Please work on requested changes.

**Review by truegilb — APPROVED** [review]
Looks good.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/543#" title="Author self-approved">resmaa</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/543#pullrequestreview-840696717" title="Approved">truegilb</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
